metpy/calc/thermo.py

```diff
@@ -362,7 +362,8 @@ def _lcl_iter(p, p0, w, t):
 @exporter.export
 @preprocess_xarray
 @check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
-def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None, dewpt_start=None):
+def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None, dewpt_start=None,
+        pick_lfc='top'):
```

Comment:
maybe call this "which"?

Reply:
I played around with this variable name too many times. `which_lfc` sounds good to me.

Reply:
I meant just "which". This follows argument to things like `shade_area`, which takes e.g. `which='positive'`